forum/qa-plugin/adventofcode-widget/src/adventofcode-page.php

```diff
@@ -76,6 +76,10 @@ private function get_html($users, $year, $leaderboard, $code)
         return $html;
     }
 
+    private function wrap_username_in_anchor($username, $userlink) {
```

Comment:
jakby chcieć się czepiać to tu klamra powinna być od nowej linii :)

Reply:
Od takich rzeczy powinien być linter :P

Reply:
Nie widzę problemu, ale jak go podepniemy na cały projekt to wszystko będzie na czerwono. Trzeba by się bawić w włączanie dla poszczególnych plików/katalogów, gdy te są dodawane lub refactorowane i jeszcze mieć nadzieję, że Q2A samo z siebie nie narzuci zbyt wielu rzeczy, które nawet w nowym kodzie linterowi się nie spodobają.

Reply:
> Od takich rzeczy powinien być prettier. :P

Uruchamianie tego dla całego projektu z kodem, który prezentuje Q2A to jak otwarcie puszki Pandory ;)

Reply:
Poprawione